README_MULTICHANNEL.md

```diff
@@ -0,0 +1,43 @@
+# MultiChannelWrapper
+
+WasatchNET has always provided support to control multiple spectrometers in 
+parallel.  However, such control is inherently manual, with all operations and
+timing left to the user.  Other than basic synchronization of the USB bus, no
+automated timing or coordination is provided across multiple devices.
+
+MultiChannelWrapper simplifies operation for developers creating multi-channel
+spectroscopic applications in which multiple spectrometers are expected to start
+acquisitions at the same time, typically via hardware triggering.
+
+Not all features will be applicable to all users, but each feature was requested
+by a particular user for a particular system, and may be found of use to others.
+
+# Proposal 
+
+We could set a "channel position" in the EEPROM, using EEPROM page 4 (allocated 
+for "customer data"), such that each spectrometer will “know” that it’s in position 
+1, 2, 8 etc (allowing user to easily reference them by channel ID in their software).
+
+- bool fanEnabled: turns fans on or off (via spectrometer with feature=fan)
+- setTriggerEnable(true) — sets all spectrometers to block on a hardware trigger 
+- startAcquisition() — sends a hardware trigger from the master (feature=trigger) 
+	to all slaves (presumably including itself)
+- getSpectra() — reads back full set of 8 spectra in a `Dictionary<channel, Tuple<double[], double[]>>`, 
+	each containing (wavelengths, intensities) for typical application measurement 
+	(blocks if triggerMeasurement has not yet been called)
```

Comment:
I know this is for a rather specific use, but I think this could potentially be used for different numbers of spectrometers than just 8, no?

Reply:
Yeah, MultiChannelWrapper should scale to any number of units (max should be ~127).  MultiChannelDemo I hardcoded to 8 because, well it's just a demo, and that has actually been a common number in the past.  It does scale well to "2", which is how many units I had for testing :-)  I'll update the README.